testsuite/cluster-test/src/tx_emitter.rs

```diff
@@ -376,6 +379,7 @@ impl SubmissionWorker {
         while !self.stop.load(Ordering::Relaxed) {
             let requests = self.gen_requests();
             let num_requests = requests.len();
+            let cur_time = Instant::now();
```

Comment:
You probably want some better name here, something like `start_time`. It is not really 'current time' on the next lines